tests/resources/storage/sample_config/bitcoin-usd-testcase.chk

```diff
@@ -28,6 +28,7 @@ spec:
     - {type: AssertIsMap, actual: {'a': 1, 'b': 2}}
     - {type: AssertIsList, actual: $Response.body.coin.exp}
     - {type: AssertIsList, actual: [1, 1, 2]}
+    - {type: AssertListContains, actual: [1, 1, 2], expected: 1}
```

Comment:
add a case for json list [sample](https://stackoverflow.com/a/3916173/5571663)

Reply:
Thanks, will update